.circleci/template.sh

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env bash
 export CIRCLECI_TEMPLATE=java-library-oss
-export PRIMARY_BRANCH=develop
+export ONLY_11=true
+export UNIT_TEST_15=true
+export UNIT_TEST_14=false
```

Comment:
I think for the final publish step, it's important the the java compiler is java 14 to ensure we don't hit the exclusion below, as we definitely want that class to appear in the final jar!

```gradle
    if (JavaVersion.current() < JavaVersion.VERSION_14) {
        excludes = ['**/Java14InputAstVisitor.java']
    }
```

So we could either use JDK=14 / JDK=15? Or perhaps we should actually diverge from the CircleCI template and do entirely independent full `./gradlew build` pipelines on both java11, java14 and maybe other higher versions too?